src/AspNet.Security.OAuth.Validation/Events/MessageReceivedContext.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Http;
 
-namespace AspNet.Security.OAuth.Validation.Events
+namespace AspNet.Security.OAuth.Validation
 {
-    public class ReceivingTokenContext : BaseControlContext
+    public class MessageReceivedContext : BaseControlContext
```

Comment:
I'm not against changing `ReceivingTokenContext` class name, but is `MessageReceivedContext` really conveying what it this event is really supposed to do? :)